### PR TITLE
[111] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by Nurmalee**
## Task: `Develop a Contact Form Page using HTML
contact-form
`
- `created a contact form with html only`

### CODE REVIEW

**Review by TundeOladejo — APPROVED** [review]

File: submissions/lawalNurudeen/html-forms/contact-form.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact-Form</title>
+</head>
+<body>
+    <form action="#" method="POST" target="_self" enctype="multipart/form-data">
```
Comment (ToluAkin) [inline submissions/lawalNurudeen/html-forms/contact-form.html:9]: @Nurmalee Remove the elements `action`, `method` and `enctype` they are not needed

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/lawalNurudeen/html-forms/contact-form.html (hunk 1 above)
Comment (Nurmalee) [reply submissions/lawalNurudeen/html-forms/contact-form.html]: Requested alterations have been applied..

File: submissions/lawalNurudeen/html-forms/contact-form.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact-Form</title>
+</head>
+<body>
+    <form action="#">
```
Comment (ToluAkin) [inline submissions/lawalNurudeen/html-forms/contact-form.html:9]: @Nurmalee Remove the attribute `action`

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/lawalNurudeen/html-forms/contact-form.html (hunk 2 above)
Comment (Nurmalee) [reply submissions/lawalNurudeen/html-forms/contact-form.html]: attribute removed

Comment (Nurmalee) [reply submissions/lawalNurudeen/html-forms/contact-form.html]: @ToluAkin @TundeOladejo I have done all required/requested changes related this pull request.. kindly process merging concerns so I can proceed to next task/topic ... Thanks

File: submissions/lawalNurudeen/html-forms/contact-form.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact-Form</title>
+</head>
+<body>
+    <form>
+        <fieldset>
+            <legend><b>CONTACT US</b></legend>
+            <div>
+                <label for="fullname">Full Name:</label><br>
+                <input type="text" name="fullname" id="fullname" size="60px" placeholder="Enter your full name here">
```
Comment (adaugochi) [inline submissions/lawalNurudeen/html-forms/contact-form.html:14]: Kindly remove the size attribute on all the input field. they are not needed

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/lawalNurudeen/html-forms/contact-form.html (hunk 3 above)
Comment (Nurmalee) [reply submissions/lawalNurudeen/html-forms/contact-form.html]: @adaugochi I have just done that..

**Review by adaugochi — APPROVED** [review]

**Pull request merged**